include/networkit/generators/BarabasiAlbertGenerator.hpp

```diff
@@ -30,7 +33,7 @@ class BarabasiAlbertGenerator final : public StaticGraphGenerator {
      * This generator implements the preferential attachment model as introduced by Barabasi and
```

Comment:
I think it is time to also change the doc string in general. Only name/reference Brandes + Batagelj and the new parallel implementation.